doc/running.rst

```diff
@@ -42,7 +42,8 @@ In order to avoid the slowdowns and warnings, users can direct the packages to c
 cache files in directories that are private to each rank by using the ``XDG_CACHE_HOME``
```

Comment:
```suggestion
cache files in directories that are private to each node by using the ``XDG_CACHE_HOME``
```
This suggestion disappeared from my original review for some reason... Anyway, is it per-rank or per-node? per-node, right?

Reply:
Sorry, did I misclick sth? In the way it is currently implemented, it is per-node.

Reply:
🤷‍♂️ Github ate it, I guess.

Reply:
"The dog ate my pull requests" 🤣